### ISSUE TITLE
[Bug]: Chapter: "Alexa Talking to Home Assistant (The Skill)" - Not working at all!

### ISSUE THREAD

**Issue by Flutzi** [keatontaylor/alexa-actions#242]
### Did you check our FAQ and Issue/PR for similar issues?

- [X] FAQ checked!
- [X] Issues/PR checked!

### What happened?

Hello.

I hope you can help me. During setup, I followed everything step by step. In the third chapter "Alexa Talking to Home Assistant (The Skill)", I worked through everything from start to finish exactly.

Now, when I try to test the skill and I enter the command "öffne actionable notifications" (in Englisch: "open actionable notifications") in the "Alexa Development Console" under "Test", it responds with: "Bei der Antwort des angeforderten Skill ist ein Problem aufgetreten." (in English: "There was a problem with the response of the requested skill!").

I have provided both my server (set up via cloudflared) and the token as described. Nevertheless, this error message appears every time after about 2-3 seconds.

Can someone tell me what's causing this? That would be really awesome. Thank you very much in advance for your help!

### Version

-

### What Alexa device are you using?

Echo

**Comment by DEADSEC-SECURITY** [comment]
I need the skill logs to be able to help. Activate debug mode and send me the log output after duing a call.Sent from my Galaxy
-------- Original message --------From: Flutzi ***@***.***> Date: 3/8/24  2:00 PM  (GMT-08:00) To: keatontaylor/alexa-actions ***@***.***> Cc: Antonio ***@***.***>, Assign ***@***.***> Subject: [keatontaylor/alexa-actions] [Bug]: Chapter: "Alexa Talking to Home
  Assistant (The Skill)" - Not working at all! (Issue #242) 
Did you check our FAQ and Issue/PR for similar issues?

 FAQ checked!
 Issues/PR checked!

Contact Details
No response
What happened?
Hello.
I hope you can help me. During setup, I followed everything step by step. In the third chapter "Alexa Talking to Home Assistant (The Skill)", I worked through everything from start to finish exactly.
Now, when I try to test the skill and I enter the command "öffne actionable notifications" (in Englisch: "open actionable notifications") in the "Alexa Development Console" under "Test", it responds with: "Bei der Antwort des angeforderten Skill ist ein Problem aufgetreten." (in English: "There was a problem with the response of the requested skill!").
I have provided both my server (set up via cloudflared) and the token as described. Nevertheless, this error message appears every time after about 2-3 seconds.
Can someone tell me what's causing this? That would be really awesome. Thank you very much in advance for your help!
Version



What Alexa device are you using?
Echo
What other Alexa device are you using?
No response
Relevant log output
No response

**Comment by Flutzi** [comment]
Thank you very very much for your answer! When I type in "öffne actionable notifications", this is the log for this command: 

_(Info: I connected my Homeassistant first with public-url (via cloudflared) and token. I changed this now to account linking without token and Nabu-Casa-URL. Both didn't work. )_

Device Log

[23:41:20:744] - Event: System.SynchronizeState
[23:41:20:755] - Event: Alexa.SmartMotion.ModeChanged
[23:41:20:765] - Event: Settings.SettingsUpdated
[23:41:20:775] - Event: System.SynchronizeState
[23:41:20:784] - Event: System.SynchronizeState
[23:41:24:454] - Event: System.SynchronizeState
[23:41:24:760] - Event: System.SynchronizeState
[23:41:26:430] - Event: System.SynchronizeState
[23:41:31:311] - Event: Text.TextMessage
[23:41:31:961] - Directive: SkillDebugger.CaptureDebuggingInfo
[23:41:32:014] - Directive: SkillDebugger.CaptureError
[23:41:32:447] - Directive: SkillDebugger.CaptureDebuggingInfo
[23:41:32:481] - Directive: SpeechSynthesizer.Speak
[23:41:32:512] - Directive: CardRenderer.RenderCard
[23:41:32:530] - Directive: SpeechRecognizer.RequestProcessingCompleted
[23:41:32:711] - Directive: SkillDebugger.CaptureError

**Comment by Flutzi** [comment]
I've now tried for an additional 6 hours and am feeling completely desperate at this point. After the twentieth backup I restored, I attempted the installation once again using the automatic installation 'AlexaActionAssistant' from Hassassistant (see: 'https://github.com/Hassassistant/AlexaActionAssistant'). The installation went smoothly. However, I'm encountering the same issue as the dozens of times before. When I run the script, I receive the message: **'There was a problem with the requested skill's response.'** 

So I assume the problem lies somewhere in the connection between the HA server and Alexa. However, I can't pinpoint exactly what the issue might be. I've tried both the Cloudflare and NabuCasa variants with tokens, as well as the variant via Account Linking.

No matter what I try, it just remains unsuccessful. Both in the Developer Console and via Home Assistant, it tells me: 'There was a problem with the requested skill's response.'

Does anyone have any idea where the problem might lie? Or is there a working alternative that you could recommend? I'm grateful for any help!

**Comment by DEADSEC-SECURITY** [comment]
Those logs look incomplete, please send new ones.

**Comment by danielbrunt57** [comment]
I changed from long lived access token to account linking and I successfully linked the skill but uttering 'Alexa, open custom notification" results in "there was a problem with the requested skill's response".  I have my Cloudflare WAF rule #1 set to skip anything going to host "alexa.<my_domain_name>" and the last item in the cloudflare event log is the account linking...

![image](https://github.com/keatontaylor/alexa-actions/assets/49846893/0a86a331-c92b-48d0-8210-fb5e7661c875)

**Comment by jross37** [comment]
I'm running into the exact same issue. Also using Cloudflare. I've gone through the instructions at least 5 different times and getting the same error "There was a problem with the requested skill's response."

**Comment by ilwalti** [comment]
Same here. Installed the latest-and-greatest 0.12.2, using italian as a language (and I get the error in Italian, that's fine!) Rebuilt some times from scratch, no joy.
This is debug log for the error...


```
{
    "header": {
        "namespace": "SkillDebugger",
        "name": "CaptureError",
        "messageId": "d7b7b350-b751-40b7-a461-xxxxxxxxxxxxxxxxx"
    },
    "payload": {
        "skillId": "amzn1.ask.skill.d8ec62a4-b9da-44de-9a06-xxxxxxxxxxxxxxxx",
        "timestamp": "2024-03-12T13:40:58.113Z",
        "dialogRequestId": "1967f08e-6bc7-4d4c-82ea-xxxxxxxxxxxxxx",
        "skillRequestId": "amzn1.echo-api.request.f08cc94d-878b-4578-acd1-xxxxxxxxxxxxx",
        "code": "SKILL_ENDPOINT_ERROR",
        "description": "Skill execution returned an exception for requestId amzn1.echo-api.request.f08cc94d-878b-4578-acd1-xxxxxxxxxx",
        "debuggingInfo": {
            "type": "SkillExecutionInfo",
            "content": {
                "invocationRequest": {
                    "endpoint": "arn:aws:lambda:eu-west-1:500099140155:function:xxxxxxxxxxxx",
                    "body": {
                        "version": "1.0",
                        "session": {
                            "new": true,
                            "sessionId": "amzn1.echo-api.session.xxxxxxxxxxxxx",
                            "application": {
                                "applicationId": "amzn1.ask.skill.xxxxxxxxxxxxxxxxx"
                            },
                            "attributes": {},
                            "user": {
                                "userId": "amzn1.ask.account.xxxxxxxxxxxxx"
                            },
                            "affiliatedResources": []
                        },
                        "context": {
                            "Extensions": {
                                "available": {
                                    "aplext:backstack:10": {}
                                }
                            },
                            "Advertising": {
                                "advertisingId": "00000000-0000-0000-0000-000000000000",
                                "limitAdTracking": true
                            },
                            "System": {
                                "application": {
                                    "applicationId": "amzn1.ask.skill.xxxxxxxxxxxxxxxxxxx"
                                },
                                "user": {
                                    "userId": "amzn1.ask.account.xxxxxxxxxxxxxxxx"
                                },
                                "device": {
                                    "deviceId": "amzn1.ask.device.xxxxxxxxxx",
                                    "supportedInterfaces": {}
                                },
                                "apiEndpoint": "https://api.eu.amazonalexa.com",
                                "apiAccessToken": "xxxxxxxxxxxxxxx"
                            }
                        },
                        "request": {
                            "type": "LaunchRequest",
                            "requestId": "amzn1.echo-api.request.f08cc94d-878b-4578-acd1-xxxxxxxxx",
                            "locale": "it-IT",
                            "timestamp": "2024-03-12T13:40:57Z",
                            "shouldLinkResultBeReturned": false
                        }
                    }
                },
                "metrics": {
                    "skillExecutionTimeInMilliseconds": 430
                }
            }
        }
    }
}
```

Any hints?

**Comment by vicky-katara** [comment]
Seeing the same issue. I also updated the Lambda code to log the bearer token. Used that bearer token with a curl command, and it works fine.

I also moved from a cloudflare-tunnel fronted URL to a direct link (via port forwarding) url -- and I see the same thing -- a 401 response.

The best bet IMO now seems like a home-assistant update has caused the response / token headers to get interpretted incorrectly?

**Comment by jross37** [comment]
> Seeing the same issue. I also updated the Lambda code to log the bearer token. Used that bearer token with a curl command, and it works fine.
> 
> I also moved from a cloudflare-tunnel fronted URL to a direct link (via port forwarding) url -- and I see the same thing -- a 401 response.
> 
> The best bet IMO now seems like a home-assistant update has caused the response / token headers to get interpretted incorrectly?

In my case I'm running HA v2024.2.0.

**Comment by danielbrunt57** [comment]
I'm running HA v2024. 3.0...

**Comment by danielbrunt57** [comment]
Here is my Cloudwatch log from my latest "alexa, open actionable notification"...
```
INIT_START Runtime Version: python:3.8.v43	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:b20481443328304931ab20faaf0929061aff7a67a0b0f40496e5e1ddec59

START RequestId: 714119e8-11a1-4377-8693-9d446afedd3c Version: 5

LAMBDA_WARNING: Unhandled exception. The most likely cause is an issue in the function code. However, in rare cases, a Lambda runtime update can cause unexpected function behavior. For functions using managed runtimes, runtime updates can be triggered by a function change, or can be applied automatically. To determine if the runtime has been updated, check the runtime version in the INIT_START log entry. If this error correlates with a change in the runtime version, you may be able to mitigate this error by temporarily rolling back to the previous runtime version. For more information, see https://docs.aws.amazon.com/lambda/latest/dg/runtimes-update.html

[ERROR] Runtime.ImportModuleError: Unable to import module 'lambda_function': No module named 'ask_sdk_core'
Traceback (most recent call last):
```

**Comment by danielbrunt57** [comment]
This wiki page has changed: https://github.com/keatontaylor/alexa-actions/wiki/Alexa-Talking-to-Home-Assistant-(The-Skill)
I was following an older cached version which said to import from a GIT URL.  I shift-reloaded that page and it now says...

> Importing the Code
> Go to the latest release of the skill and download the zip file called AlexaActionsNoBinary.zip
> In the alexa developer console open the code editor of the skill
> Click in Import Code
> Select the zip file you downloaded
> Select the lambda folder and click import. **Note: ** If you get an error saying that you're over the number of files limit, try importing partially and then repeat to import the rest.
> (Optional) You can unselect any folder that ends with .dist-info
> Troubleshooting
> If you're skill doesn't work you might need to import with binaries so try again but with the zip file named AlexaActionsWithBinary.zip

but I cannot find either of those two zip files...

**Comment by danielbrunt57** [comment]
After another delete-and-recreate skill, I now have this error in cloudwatch log...
```
[ERROR] Runtime.ImportModuleError: Unable to import module 'lambda_function': No module named 'isodate'
Traceback (most recent call last):
```

**Comment by danielbrunt57** [comment]
Yet another fresh attempt and now no cloudwatch log entry at all but still have _There was a problem with the requested skills response._

**Comment by RAW297** [comment]
@keatontaylor 

is there any movement on finding a fix to this issue please?

It seems there a many people all experiencing the same issue.

Happy to help with any logs or testing you need.

See [here](https://community.home-assistant.io/t/alexa-actionable-notifications-skill-error/702522/2) for logs I uploaded recently.

if you need other logs please direct me where to find them and will happily provide and test as required, have some free time Friday to test if needed.

`
INIT_START Runtime Version: python:3.8.v43	Runtime Version ARN: arn:aws:lambda:eu-west-1::runtime:b20481443932830049531ab20faaf09295061aff7a67a0b0f40496e5e1ddec59


LAMBDA_WARNING: Unhandled exception. The most likely cause is an issue in the function code. However, in rare cases, a Lambda runtime update can cause unexpected function behavior. For functions using managed runtimes, runtime updates can be triggered by a function change, or can be applied automatically. To determine if the runtime has been updated, check the runtime version in the INIT_START log entry. If this error correlates with a change in the runtime version, you may be able to mitigate this error by temporarily rolling back to the previous runtime version. For more information, see https://docs.aws.amazon.com/lambda/latest/dg/runtimes-update.html


[ERROR] Runtime.ImportModuleError: Unable to import module 'lambda_function': No module named 'isodate'
Traceback (most recent call last):
`

**Comment by danielbrunt57** [comment]
I just got it working.  I was a dumb-dumb!
I found & downloaded [AlexaActionsNoBinaryLinuz.zip](https://github.com/keatontaylor/alexa-actions/releases), imported the code into my lambda, deployed it and it's working.

![image](https://github.com/keatontaylor/alexa-actions/assets/49846893/e90c301a-5382-4777-b1e1-d2965d1a5172)

![image](https://github.com/keatontaylor/alexa-actions/assets/49846893/39aab5d8-8125-4433-a24f-666fe9c0e9f7)

**Comment by Flutzi** [comment]
> d [AlexaActionsNoBinaryLinuz.zip](https://github.com/keatontaylor/alexa-actions/releases)

Wow! That was the solution! Thank you very much! Finally, the step worked! 

My mistake was that I couldn't find the file **"AlexaActionsNoBinaryLinux.zip"** at all. Instead, during the skill installation, I inserted the GitHub link or uploaded the ZIP file (However, the ZIP file that can be downloaded from the code page is named "alexa-actions-master"). **Maybe it's time to tidy up the wiki here and make the description a bit clearer.** 

Otherwise, it works great, even with Cloudflared. If you encounter any issues with Cloudflared, try logging in there, select your URL, then go to "Security" > "Bots" on the left, and disable "Bot Fight Mode". That might help some users. 

Again, a big thank you to you @danielbrunt57! You were a huge help!

**Comment by danielbrunt57** [comment]
> @danielbrunt57
> 
> Wow! That was the solution! Thank you very much! Finally, the step worked!
> 
> My mistake was that I couldn't find the file **"AlexaActionsNoBinaryLinux.zip"** at all. Instead, during the skill installation, I inserted the GitHub link or uploaded the ZIP file (However, the ZIP file that can be downloaded from the code page is named "alexa-actions-master"). **Maybe it's time to tidy up the wiki here and make the description a bit clearer.**
> 
> Otherwise, it works great, even with Cloudflared. If you encounter any issues with Cloudflared, try logging in there, select your URL, then go to "Security" > "Bots" on the left, and disable "Bot Fight Mode". That might help some users.
> 
> Again, a big thank you to you @danielbrunt57! You were a huge help!

I worked on this for days and like you I also use Cloudflare and was making the same mistakes until I woke up this morning and found the elusive zip file with the missing/required components (which is all new this month I think)!  I spent hours yesterday with python/pip trying to put together what was missing.  I will post something/somewhere about my Cloudflare tunnel setup to keep the baddies out yet allow me on my phone or my laptop to get to my HA and Vaultwarden and also permit the Home Assistant Alexa skill (which serves double duty to permit Actionable Notifications).  I'm also going to do a PR to update the wiki with a minimum of a clickable link to the releases area to download the file:
> 1. Go to the latest release of the skill and download the zip file called [AlexaActionsNoBinary.zip](https://github.com/keatontaylor/alexa-actions/releases)

**Comment by RAW297** [comment]
@danielbrunt57

You legend!  That has now worked for me too!

@keatontaylor @DEADSEC-SECURITY 

Could we see the installation documentation updated to include links to where to find the relevant downloads etc.  Speaking for myself, i'm no devleoper and was simply following the instructions slightly blind as wasn't able to find the Zip files even after searching in Github.

Thanks again for sharing @danielbrunt57.

**Comment by danielbrunt57** [comment]
Wiki updated.

![image](https://github.com/keatontaylor/alexa-actions/assets/49846893/cebc3404-b44d-489f-8b42-bc2352aa7647)

![image](https://github.com/keatontaylor/alexa-actions/assets/49846893/60065feb-2638-48b4-b8ac-d7ef0c0e5930)

**Comment by danielbrunt57** [comment]
Now to get the authors of ALL of the 3rd party how-to's & videos updated........

**Comment by jross37** [comment]
> Wiki updated.
> 
> ![image](https://private-user-images.githubusercontent.com/49846893/313192815-cebc3404-b44d-489f-8b42-bc2352aa7647.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA1MDY0MzksIm5iZiI6MTcxMDUwNjEzOSwicGF0aCI6Ii80OTg0Njg5My8zMTMxOTI4MTUtY2ViYzM0MDQtYjQ0ZC00ODlmLThiNDItYmMyMzUyYWE3NjQ3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE1VDEyMzUzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNlNDAwMGMwYzcxNWU0NTMxZmVjOGE5Y2RkMDFkMmViZGRmZDQ3NzBlM2Y5NDhmZjVhZTJhNDdiMzcxNTAwNzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3I9lOEYFHCjE24gI0XWdtPmT64e0hDn0Xk62FJQzff8)
> 
> And new image reflecting what you would see after import...
> 
> ![image](https://private-user-images.githubusercontent.com/49846893/313192937-60065feb-2638-48b4-b8ac-d7ef0c0e5930.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA1MDY0MzksIm5iZiI6MTcxMDUwNjEzOSwicGF0aCI6Ii80OTg0Njg5My8zMTMxOTI5MzctNjAwNjVmZWItMjYzOC00OGI0LWI4YWMtZDdlZjBjMGU1OTMwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE1VDEyMzUzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEzYjk5NTcxZGU0ODMzYjNjNjc4ZDc3ODgxNzMwNmZmNzU1ODQwNmZjMzRmYmRkOTNlNmU3M2EzYWUyODllMzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9OF6U4wrh6infRceT4pkeR9Iz7E3c90GpLgUtpx6Iwg)

Thank you! This is much clearer. Originally it sounded like the code import was more optional -- other walk throughs (and the wiki) made it seem like all you needed to was copy in the 5 files or so. Looks like there's a lot more that comes in with the import. 

Appreciate the help from everyone!

**Comment by danielbrunt57** [comment]
@keatontaylor @DEADSEC-SECURITY (and everyone else) - Your welcome!!!

**Comment by Flutzi** [comment]
@danielbrunt57 

Thank you again for your crucial hint. I was finally able to successfully connect Alexa with HA, and now Alexa is also asking me: „Did you prepare your coffee?“ So, I’m getting the appropriate question. But it doesn’t matter what I answer, Alexa unfortunately always says: “You just triggered AMAZON.FallbackIntent.” Can anyone help? That would be great! 

Have you happened to stumble upon the same issue and have any advice?

**Comment by DEADSEC-SECURITY** [comment]
Thank you @danielbrunt57

**Comment by danielbrunt57** [comment]
> Alexa unfortunately always says: “You just triggered AMAZON.FallbackIntent.”

My first thought upon reading this is your skill likely only lists the 4 very basic intents (Cancel. Help, Stop & NavigateHome).
There should be 11 and your JSON Editor should look like this...

![image](https://github.com/keatontaylor/alexa-actions/assets/49846893/10c59585-f64f-4145-b222-57890679cdf8)

and if it doesn't then there was a missed/incorrect step in the building a new skill.

**Comment by Flutzi** [comment]
@danielbrunt57 

Oh man, you're really a god. Thank you very much, that's exactly where the problem lies. Strangely, I only had 6 intents listed in the JSON editor. Even when I rebuilt the skill, only 6 intents were to be found. I have now found the corresponding JSON file here on GitHub and manually inserted the missing lines. Now it finally works! Thank you so much for helping me and others so much! You're truly a selfless gentleman. I wish you a beautiful and fulfilling life, stay as you are :)

**Comment by danielbrunt57** [comment]
> Thank you @danielbrunt57

@DEADSEC-SECURITY @Flutzi 
I just stepped through the wiki instructions carefully to create a new Test skill and was unable to follow the instructions step-by-step having to guess what to do next and ended up with only the basic default 6 slot types rather than the custom skill's required 11 slot types.

![image](https://github.com/keatontaylor/alexa-actions/assets/49846893/1972a820-f760-4dd2-abb2-7cf8c3c8f81b)

There's a step missing in between...

> Finally select your "Hosting region".
>
> Screenshot of region selection
> Click the button "Next" in the upper right hand corner.

and here...

> Importing the Code
> 1. Go to the latest release of the skill and download the zip file called [AlexaActionsNoBinary.zip](https://github.com/keatontaylor/alexa-actions/releases)

That step is to import the skill from https://github.com/keatontaylor/alexa-actions.git:

![image](https://github.com/keatontaylor/alexa-actions/assets/49846893/bfbb4086-d6ca-4fe8-a5e4-7d1ee51b2da5)

If we don't do that then we end up missing the custom JSON file as I and others have done!

I'll modify the wiki...

**Comment by danielbrunt57** [comment]
> Thank you @danielbrunt57

@DEADSEC-SECURITY @Flutzi 
I just stepped through the wiki instructions carefully to create a new Test skill and was unable to follow the instructions step-by-step having to guess what to do next and ended up with only the basic default 6 slot types rather than the custom skill's required 11 slot types.

![image](https://github.com/keatontaylor/alexa-actions/assets/49846893/1972a820-f760-4dd2-abb2-7cf8c3c8f81b)

There's a step missing in between...

> Finally select your "Hosting region".
>
> Screenshot of region selection
> Click the button "Next" in the upper right hand corner.

and here...

> Importing the Code
> 1. Go to the latest release of the skill and download the zip file called [AlexaActionsNoBinary.zip](https://github.com/keatontaylor/alexa-actions/releases)

That step is to import the skill from https://github.com/keatontaylor/alexa-actions.git:

![image](https://github.com/keatontaylor/alexa-actions/assets/49846893/bfbb4086-d6ca-4fe8-a5e4-7d1ee51b2da5)

If we don't do that then we end up missing the custom JSON file as I and others have done!

I'll modify the wiki...

**Comment by danielbrunt57** [comment]
@DEADSEC-SECURITY
I've finished modifying the Wiki at https://github.com/keatontaylor/alexa-actions/wiki/Alexa-Talking-to-Home-Assistant-(The-Skill).  I'd appreciate someone (Antonio?) stepping through it as a total noob to see if it's accurate now and whether there any errors, omissions, typos, etc.!!!  Thanks.

**Comment by legolas0802** [comment]
I followed all the instructions in the post, the updated guide, tried both new zip files, but I still have the same problem

**Comment by danielbrunt57** [comment]
> I followed all the instructions in the post, the updated guide, tried both new zip files, but I still have the same problem

Retry but use [my forked repo](https://github.com/danielbrunt57/alexa-actions.git) to import the skill from and you can skip the ZIP upload step as I included those files in the repo...

**Comment by Seppel62** [comment]
I can not do it. no matter what I do Error "There was a problem responding to the requested skill"
Alexa_media_player works. Alexa can switch devices from HA, text from HA arrives at Alexa devices. "SKILL_ENDPOINT_ERROR" is displayed as an error in Alexa Developer

**Comment by flynbvr** [comment]
Ive started with some videos and attempted the deployment using what I now understand to be as the old method/process. I tried multiple times before I found this thread. Then I attempted the deployment 4 more times using the current wiki information including the edits/updates from this thread but I still get the same "problem responding". An unfortunate waste of time at this point. The rabbit hole is so deep Im not sure when Ill see light again.
